wrappers/README.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines for Libindy Wrappers
```

Comment:
As this readme acutally contains Guidelines for wrappers developer, may be it's better to don't use base `README.md` filename, but rename it to something like `guidelines.md` ?